api-docs/autogen/api/v2_users_user_openid_messages.post.md

```
@@ -0,0 +1,436 @@
+
+|名称|类型|必填|描述|
+|---|---|---|---|
+|msg\_type|integer|否|消息类型。决定哪个内容字段生效: 0=纯文本(content) 2=Markdown(markdown) 3=ARK(ark) 6=输入中状态（input\_notify) 7=富媒体(media)|
+|content|string|否|文本内容。msg\_type=0 时为全文 注意: 传了 markdown 后此字段必须为空|
+|markdown|[MessageMarkdown](#schema-messagemarkdown)|否|Markdown 消息。msg\_type=2 时必填 注意: 填写此字段后 content/ark 必须全为空|
+|ark|[MessageArk](#schema-messageark)|否|结构化卡片消息。msg\_type=3 时填写，未对外开放，需要申请白名单|
+|keyboard|[Keyboard](#schema-keyboard)|否|内嵌键盘。短形式只传 id，长形式传 content.rows|
+|msg\_id|string|否|被动回复的消息 ID。从 C2C\_MESSAGE\_CREATE 等事件的 d.id 获取，5 分钟内有效|
+|event\_id|string|否|被动回复的事件 ID。从事件最外层的id获取。与 msg\_id 二选一，支持事件："INTERACTION\_CREATE"、"C2C\_MSG\_RECEIVE"、"FRIEND\_ADD"|
+|msg\_seq|integer|否|回复消息的序号，与 msg\_id 联合使用，避免相同消息 id 回复重复发送，不填默认是 1。相同的 msg\_id + msg\_seq 重复发送会失败。|
+|media|[MediaInfo](#schema-mediainfo)|否|富媒体消息。msg\_type=7 时填写，file\_info 来自 /v2/groups/{group\_openid}/files|
```

markdownlint-cli2 (0.23.0)

[warning] Link fragments should be valid

(MD051, link-fragments)